stats/view/worker_commands.go

```diff
@@ -104,6 +104,9 @@ func (cmd *unregisterFromViewReq) handleCommand(w *worker) {
 			vi.clearRows()
 		}
 		w.unregisterView(name)
+		if measure := w.measures[vi.view.Measure.Name()]; measure != nil {
+			delete(measure.views, vi)
+		}
```

Comment:
Just like reference to view is added in measure ref during tryRegisterView, removal should also be in unregisterView func.

Reply:
Done.